resources/startup.sh

```diff
@@ -22,6 +22,8 @@ export OPENLDAP_SUFFIX="dc=cloudogu,dc=com"
 
 ulimit -n ${OPENLDAP_ULIMIT}
 
+######
+
```

Comment:
What are these lines about? Refactoring artefacts?

Reply:
I think they were there to mark the functions section which was removed. So i removed them.